content/about/globalteam/index.en.md

```diff
@@ -0,0 +1,125 @@
+
+## Commitment
+
+As R-Ladies is a purely volunteer run organisation, we respect and encourage your
+need to have vacations, free-time and focus on other aspects of your life. The 
+hours spent on R-Ladies work should happen when you are truly available to do them. 
+We do ask those that are seeking to join the Global Team commit to one year of service 
+(of course, we would understand if life interrupts this commitment), and an estimate
+of the weekly time commitment is 1 to 4 hours a week. In addition, the needs of
+the specific roles may vary: some volunteers work on a weekly, monthly, or quarterly
+basis due to natural workload fluctuations.
+
+If a volunteer needs to temporarily pause their responsibilities for any reason
+(e.g., work, health, or vacation) please let the leadership team know so that we
+may fill your role during your absence.
+
+If a volunteer is ready to exit the Global Team for any reason, please let the leadership
+team know. If you are capable, we would appreciate it if you have the capacity to on-board a new
+team member to your role; but if you are unable, we understand.
+
+We suspect that after service over time in a specific role you may be ready to
+try a new role, if you wish to continue on the Global Team. If you have reached that
+point, please let the leadership team know so that we may re-assign you to role
+that aligns with your current interests.
+
+## Communication
+
+Our primary communication channel is Slack, and the common language we communicate
+in is English. We hope for Global Team members to regularly check Slack; if the 
+team member is not present on Slack, then we require them to be responsive via email.
+
+## Operations
+
+Individual teams within the Global Team may operate relatively independently
+from the Global Team or may work more closely with the Global Team. Individual
+teams may establish their own work preferences or set their own small team meetings.
+In addition, many, but not all, members of the Global Team serve on multiple 
+individual teams. Lastly, belonging to a specific team does not mean your contributions
+or communications are limited to that team. Individual teams often solicit and
+receive feedback or assistance from other members of the Global Team.
+
+## Joining the Global Team
+
+The Global Team regularly needs to fulfill roles. This may be accomplished by 
+asking on the organizers or community slack if anyone has capacity to fulfill a 
+position, via a more formal call for volunteers via a News announcement on the 
+R-Ladies website (such as the one from [2023-04-11](https://rladies.org/news/2023-04-11-global-team-recruiting/)),
+or via individual volunteers reaching out.
+
+In selecting volunteers, we are looking for demonstrated:
+1. alignment with the values of the Global Team, and
```

Comment:
I feel like alignment with core values could feel vague to some people in terms of what that looks like but I am not coming up with a good concrete example we could add. I am going to think more about this.

Reply:
i am resolving this, but we can come back to ammend later if needed.